contracts/allowlist/OperatorAllowlistUpgradeable.sol

```diff
@@ -0,0 +1,189 @@
+// Copyright Immutable Pty Ltd 2018 - 2023
```

Comment:
This contract has a lot in common with OperatorAllowlist. The two contracts should inherit from a common OperatorAllowlistBase that contains the common code and is written to be used by both of them.

Reply:
We will delete the OperatorAllowlist base contract once the repo's dependencies are fixed